### PR TITLE
Change KLASS to SAWS

### PR DESCRIPTION
**Pull request by mrhinkle**
Change KLASS to SAWS

[Rendered page](https://github.com/triggermesh/aws-event-sources/tree/mrhinkle-patch-1#readme)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![TriggerMesh Knative Lambda Sources](./images/klass.png "TriggerMesh Knative Lambda Sources")
 
-**What:** Knative Lambda Sources (KLASS) are Knative event sources for AWS services.
+**What:** Sources for Amazon Web Services(Sources) are event sources for AWS services.
```
Comment (sebgoa) [inline README.md:5]:
```suggestion
**What:** Sources for Amazon Web Services (SAWS) are event sources for AWS services.
```

**Comment by sebgoa** [comment]
Do we have a new logo ?

**Comment by mrhinkle** [comment]
@sebgoa I thought I made a pull request for the logo
sources_for_amazon_web_services

**Comment by sebgoa** [comment]
you need to stick the .png in the ./images folder so that it is part of the PR.

**Comment by antoineco** [comment]
We may also want to

1. Push a logo with a slightly better resolution, this one is quite tiny.
2. Have a [square version](https://github.com/triggermesh/aws-event-sources/blob/bb9df24537eba8ea87bf67774b1895ef9923e76e/images/klass-logo.png) for @sameersbn, who uses it in the Helm Chart.

**Pull request merged**